### ISSUE TITLE
tkinter variable option for Slider, label and entry widget

### ISSUE THREAD

**Issue by IshanJ25** [TomSchimansky/CustomTkinter#17]
```
tk.Entry(root, textvariable=txt_var)
tk.Scale(root, from_=f, to=t, variable=int_var)
```

how to achieve this setting in ctk?

**Comment by IshanJ25** [comment]
I want a function triggered whenever the value is changed in either text box or scale

`txt_var_r = tk.StringVar()
txt_var_g = tk.StringVar()
txt_var_b = tk.StringVar()

txt_var_r.trace('w', lambda name, index, mode, var=txt_var_r: text_changed)
txt_var_g.trace('w', lambda name, index, mode, var=txt_var_g: text_changed)
txt_var_b.trace('w', lambda name, index, mode, var=txt_var_b: text_changed)

txt_var_hex = tk.StringVar()
txt_var_hex.trace('w', lambda name, index, mode, var=txt_var_hex: hex_changed)

int_var_r = tk.IntVar()
int_var_g = tk.IntVar()
int_var_b = tk.IntVar()

int_var_r.trace('w', lambda name, index, mode, var=int_var_r: slider_changed)
int_var_g.trace('w', lambda name, index, mode, var=int_var_g: slider_changed)
int_var_b.trace('w', lambda name, index, mode, var=int_var_b: slider_changed)`

here `slider_changed', 'text_changed` and 'hex_changed` are function i need to be called

code for spawning widgets:
`
frame = ctk.CTkFrame(root, width=400, height=400, corner_radius=20)
    frame.place(x=0, y=0)

    ctk.CTkSlider(frame, from_=0, to=25, variable=int_var_r,
                  fg_color=color['light1'], button_color=color['light2'], progress_color=color['red'],
                  ).place(x=20, y=180, width=600)

    ctk.CTkSlider(frame, from_=0, to=25, variable=int_var_g,
                  fg_color=color['light1'], button_color=color['light2'], progress_color=color['green'],
                  ).place(x=20, y=220, width=600)

    ctk.CTkSlider(frame, from_=0, to=25, variable=int_var_b,
                  fg_color=color['light1'], button_color=color['light2'], progress_color=color['blue'],
                  ).place(x=20, y=260, width=600)
`

**Comment by TomSchimansky** [comment]
Tkinter variables are not integrated yet, just because I never used them before, but I will try to do it in the next few days.

**Comment by TomSchimansky** [comment]
Tkinter variables are now available as of version 2.0 on pip.
